src/Library/demos/Spin Button/main.js

```diff
@@ -0,0 +1,31 @@
+const hours = workbench.builder.get_object("hours");
+const minutes = workbench.builder.get_object("minutes");
+
+hours.text = "00";
+minutes.text = "00";
+
+hours.connect("value-changed", () => {
+  console.log(tellTime(hours, minutes));
+});
+
+hours.connect("output", () => {
+  const value = hours.adjustment.value;
+  const text = value.toString().padStart(2, "0");
+  hours.text = text;
+  return true;
+});
+
+minutes.connect("output", () => {
+  const value = minutes.adjustment.value;
+  const text = value.toString().padStart(2, "0");
+  minutes.text = text;
+  return true;
+});
+
+minutes.connect("value-changed", () => {
+  console.log(tellTime(hours, minutes));
+});
+
```

Comment:
Oo that's a great idea!

Reply:
I added this
```js
minutes.connect("wrapped", () => {
  hours.spin(Gtk.SpinType["SPIN_STEP_FORWARD"], 1);
});
```
But the only problem with this is that ``::wrapped`` will get triggered when the minute button is decremented too but it'll increment the hours button. Since you said nothing fancy, this works ig?

Reply:
Hmm, good point. I guess you'd have to track the pre-wrapped value, which would kind of distract from the SpinButton.

Maybe just add a comment, pointing out this only works in one direction and the developer can take it from there.

Reply:
Okay 👍🏾